utils/main.moustache

```diff
@@ -0,0 +1,316 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# deploy-cloud-functions
```

Comment:
I'm not in favor of moving this entire thing into a template file. It makes external contributions difficult. Instead, could we render this list to another file in `docs/` and then just link to it from the main README instead?

Reply:
Yep, I definitely understand. Had not thought about the complications of using a template file. Will rework with the `docs/` approach instead.